<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageObjectVolumeSettingsWidget</class>
 <widget class="QWidget" name="ImageObjectVolumeSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="enableVolumeRenderingCheckBox">
     <property name="text">
      <string>Enable Volume Rendering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showClippingWidgetCheckBox">
     <property name="text">
      <string>Show Clipping Widget</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="shadingCheckBox">
     <property name="text">
      <string>Enable Shading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="windowLevelFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="levelLabel">
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="levelSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="windowLabel">
        <property name="text">
         <string>Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="windowSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="sampleDistanceFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="autoSampleDistanceCheckBox">
        <property name="text">
         <string>Auto sample distance</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="sampleDistanceLabel">
          <property name="text">
           <string>SampleDistance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sampleDistanceSpinBox">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="shadingParamsFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ambiantLabel">
          <property name="text">
           <string>Ambiant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="ambiantSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="specularLabel">
          <property name="text">
           <string>Specular</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="specularSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="diffuseLabel">
          <property name="text">
           <string>Diffuse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="diffuseSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="specularPowerLabel">
          <property name="text">
           <string>Specula Pow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="specularPowerSpinBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="vtkQtPiecewiseFunctionWidget" name="scalarOpacityFunctionWidget" native="true">
     <property name="toolTip">
      <string>Scalar Opacity Transfer Function</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="vtkQtColorTransferFunctionWidget" name="colorFunctionWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Color Transfer Function</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>501</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vtkQtPiecewiseFunctionWidget</class>
   <extends>QWidget</extends>
   <header>vtkQtPiecewiseFunctionWidget.h</header>
  </customwidget>
  <customwidget>
   <class>vtkQtColorTransferFunctionWidget</class>
   <extends>QWidget</extends>
   <header>vtkQtColorTransferFunctionWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
